### PR TITLE
Let ethereals have gender

### PR DESCRIPTION
**Pull request by plsleavemealon**
## About The Pull Request
Removes the trait that prevents ethereals from having gender.

## Why It's Good For The Game

Because being unable to customize how your character sounds is really dumb, and also forcing people's characters to always be addressed in a certain manner is equally dumb.

## Changelog

:cl:
del: Removed the agender trait from ethereals.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/ethereal.dm [hunk 1]
```diff
@@ -11,11 +11,10 @@
 	siemens_coeff = 0.5 //They thrive on energy
 	payday_modifier = 1.0
 	inherent_traits = list(
-		// TRAIT_NO_UNDERWEAR, // SKYRAT EDIT - LET THEM WEAR PANTIES
+		// TRAIT_NO_UNDERWEAR, // SKYRAT EDIT - LET THEM WEAR PANTIES //BUBBER EDIT - LET THEM HAVE GENDER
```
Comment (StrangeWeirdKitten) [inline code/modules/mob/living/carbon/human/species_types/ethereal.dm:14]: Can you put this commit behind TRAIT_AGENDER?

``//TRAIT_AGENDER - BUBBER EDIT``

Comment (plsleavemealon) [reply code/modules/mob/living/carbon/human/species_types/ethereal.dm:14]: I am not sure what you mean, I removed TRAIT_AGENDER from that list.

The trait still exists though, its used again for the lustrous variant of ethereals, the pirate variant, which I didn't touch.

Comment (plsleavemealon) [reply code/modules/mob/living/carbon/human/species_types/ethereal.dm]: Do you mean like this?

**Comment by StrangeWeirdKitten** [comment]
Yeah that's fine. I just need to know exactly what's being removed from lists.

**Review by StrangeWeirdKitten — APPROVED** [review]

**Comment by nikothedude** [comment]
shame, it added flavor, having the implication of a genderless society is something fun to play with

**Pull request merged**